cmd/buildah/addcopy.go

```diff
@@ -126,3 +138,18 @@ func addCmd(c *cobra.Command, args []string, iopts addCopyResults) error {
 func copyCmd(c *cobra.Command, args []string, iopts addCopyResults) error {
 	return addAndCopyCmd(c, args, "COPY", false, iopts)
 }
+
+func parseDockerignore(ignoreFile string) ([]string, error) {
+	var excludes []string
+	ignore, err := ioutil.ReadFile(ignoreFile)
+	if err != nil {
+		return excludes, err
```

Comment:
Should this error be wrapped?

Reply:
No, ioutil.ReadFile brings back a good error messages including the ignorefile.  (Previous version had it wrapped, and all it did was stutter.

Reply:
Some of those are probably candidates for `errors.WithStack()`.